src/Microsoft.Azure.SignalR.Common/Exceptions/AzureSignalRAccessTokenNotAuthorizedException.cs

```diff
@@ -22,11 +23,22 @@ public AzureSignalRAccessTokenNotAuthorizedException(string message) : base(mess
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="AzureSignalRAccessTokenNotAuthorizedException"/> class.
+        /// Obsolete, <see cref="AzureSignalRAccessTokenNotAuthorizedException(TokenCredential, Exception)"/>.
```

Comment:
then the other ctor shouldn't be internal?

Reply:
The customer shouldn't construct this exception by himself.